### ISSUE TITLE
Support real global batch normalisation

### ISSUE THREAD

**Issue by luomai** [lsds/KungFu#281]
Tensorflow 2.2 supports the following feature:

> Support added for global sync BatchNormalization by using the newly added tf.keras.layers.experimental.SyncBatchNormalization layer. This layer will sync BatchNormalization statistics every step across all replicas taking part in sync training.

It would be great if KungFu can support it as well.

**Comment by luomai** [comment]
@lgarithm @marwage @fertakis

**Comment by luomai** [comment]
I recently had a discussion with my friend in Google. It seems like people are moving to use Group Normalisation or other Normalisation approaches instead of using Batch Norm because Batch Norm is indeed difficult synchronise in a distributed setting. Also, Batch Norm is difficult to be applied in a RL scenario because it out-weight the samples in an online trajectory.
